CONTRIBUTING.md

```diff
@@ -237,6 +242,30 @@ body:
 For additional information on using the `dep` tool, see
 "[Performing vendoring for the Kata Containers project](https://github.com/kata-containers/community/blob/master/VENDORING.md)".
 
+### Stable branch backports
+
+Kata Containers maintains a number of stable branch releases. Bug fixes to the
+master branch are selectively applied to ('backported') these stable branches.
```

Comment:
This still reads slightly strangely to me. I'd consider:

> Bug fixes to the `master` branch are selectively applied (or 'backported') to these stable branches.

Reply:
heh, I don't like the 'or' in the brackets - to mean that reads as if it is an alternative, not a pseudonym.
Re-reading, what if I slide the 'to' beyond the brackets and add an 'aka', so we get:
> are selectively applied (a.k.a. 'backported') to these stable

Any better @jodh-intel ?

Reply:
that would work too although "a.k.a." isn't going to be as easily understood to non-native readers as "or" I think :)

Reply:
I stuck your 'or' (or is that your 'oar'? :-) ) in @jodh-intel